core/testing.go

```diff
@@ -29,7 +29,11 @@ func StartTestNode(ctx context.Context, t *testing.T, app types.Application, cfg
 		options.SpecificConfig = cfg
 	})
 	t.Cleanup(func() {
-		rpctest.StopTendermint(nd)
+		err := nd.Stop()
```

Comment:
@distractedm1nd can you doc why we do this?

Reply:
Hey, think thats from an older commit, `rpctest.StopTendermint` works again :D !

Reply:
shipit